docs/metrics.md

```diff
@@ -24,6 +24,48 @@ of the last second of activity, whereas the 1 minute intervals are used for
 reporting values to an external database in which case the values reported are
 all reflective of the last minute of activity.
 
+### Default Metrics
+
+Colossus makes several metrics available in a default service, they are:
+
+* *service name*/system/fd_count
```

Comment:
It would be good to explain what each of these means and which part of colossus they come from (and maybe tags that can be found on them?)